tldraw/tldraw
# fix(ui): ensure context menu and menus close correctly

This PR fixes a bug that causes menus not to close correctly when open.

### Change type


### Test plan

1. On mobile, open the menu.
2. Tap the canvas—it should close the panel.
3. Open the mobile style panel.
4. Tap the canvas—it should close the panel.
5. Open the mobile style panel.
6. Tap the mobile style panel button—it should close the panel.
7. On mobile, long press to open the context menu


### Release notes

- Fixes a bug where menus and the context menu would not close correctly on interaction.
